Tasks/Common/webdeployment-common-v2/xmlvariablesubstitutionutility.ts

```diff
@@ -125,7 +125,7 @@ export function substituteXmlVariables(configFile, tags, variableMap, parameterF
         }
         for(var xmlNode of nodes) {
             if(varUtility.isObject(xmlNode)){
-                tl.debug("Processing substitution for xml node : " + xmlNode.name);
+                console.log(tl.loc('SubstitutionForXmlNode' , xmlNode.name));
```

Comment:
https://github.com/microsoft/azure-pipelines-tasks/blob/56378f39cc0ab992b7b65a96625484b877f00fcb/Tasks/Common/webdeployment-common-v2/xmlvariablesubstitutionutility.ts#L159-L162

Can you also add these statements as console? This would say whether the config file is updated or not